SparkPost/2web2ui
# AC-1789: Logout request 400s fails for returning users

[AC-1789](https://sparkpost.atlassian.net/browse/AC-1789)

### What Changed
- Moved use of `SparkpostApiError` from `sparkpostApiRequest` to `sparkpostErrorHandler` interceptor
- Moved file location of `SparkpostApiError` and `ZuoraApiError` from `src/actions/helpers` to `src/helpers`
- Commented out the network error test for `useSparkPostQuery` — I was unable to find a reasonable solution

Documented my findings here — https://sparkpost.atlassian.net/l/c/yohoPn2z

### How To Test

You will need to pull latest `charger`.

 - Follow steps in bug ticket to confirm it was fixed
 - Need to confirm other axios instances and useSparkPostQuery error handling works correctly with SparkpostApiError's
